### PR TITLE
Add gpu-1.8 jobs

### LINKED ISSUE #4678: Create a pool of gcp projects with gpu quota
Currently, for each GPU test suite, we specify a particular gcp project that has gpu quota. Other (non-gpu) tests use projects from a pool. Let's create a pool of projects with gpu quota and make gpu test suites use that pool, so that we don't have to create new projects for new gpu test suites.

Here's a list of GCP projects that have gpu quota:
```
k8s-jkns-e2e-gce-plugin-gpus
k8s-jkns-e2e-gce-gpus
k8s-jkns-e2e-gce-gpus-1-7

k8s-jkns-e2e-gke-plugin-gpus
k8s-jkns-e2e-gke-gpus
k8s-jkns-e2e-gke-gpus-1-7
ci-kubernetes-e2e-gke-gpu    # this is not a typo
```

The first three are setup for GCE. The last four are setup for GKE.

**AI 1) Make all of them work for GKE as well.**

The suites that need to use these projects are:
```
ci-kubernetes-e2e-gce-device-plugin-gpu
ci-kubernetes-e2e-gce-gpu
ci-kubernetes-e2e-gce-gpu-1-7
ci-kubernetes-e2e-gke-device-plugin-gpu
ci-kubernetes-e2e-gke-gpu
ci-kubernetes-e2e-gke-gpu-1-7
```

**AI 2) Make the above suites use the projects from the gpu pool instead of using specific projects.**

We want to add 4 more suites that will use projects from the gpu pool (these are for 1.8):
```
ci-kubernetes-e2e-gce-gpu-1-8 # every 2h
ci-kubernetes-e2e-gke-gpu-1-8 # every 2h
ci-kubernetes-e2e-gce-device-plugin-gpu-1-8 # every 2h
ci-kubernetes-e2e-gke-device-plugin-gpu-1-8 # every 24h
```

**AI 3) Create these suites and make them use the gpu project pool.**

There are two other suites:
```
pull-kubernetes-e2e-gce-gpu (runs on all PRs)
pull-kubernetes-e2e-gke-gpu (runs on PRs by demand)
```

These suites current use `k8s-jkns-pr-gce-gpus` (high quota) and `k8s-jkns-e2e-gke-gpus` (shared with CI) currently.

**A1 4) Create a `k8s-jnks-pr-gke-gpus` project to use with `pull-kubernetes-e2e-gke-gpu` till we create a pool for gpu PR tests.**

### PR DESCRIPTION
**Pull request by krzyzacy**
fixes #4678 

new jobs uses the gpu project pool, and I will migrate existing jobs over later on.

/assign @mindprince @BenTheElder

### CODE REVIEW

**Comment by mindprince** [comment]
Thanks for setting this up!

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -3506,16 +3526,24 @@ dashboards:
   dashboard_tab:
   - name: gce-device-plugin-gpu
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu
+  - name: gce-device-plugin-gpu-stable1
```
Comment (mindprince) [inline testgrid/config/config.yaml]: Let's call this `gce-device-plugin-gpu-1-8` like the others.

Comment (krzyzacy) [reply testgrid/config/config.yaml]: whoops, slipped, will patch up

Comment (krzyzacy) [reply testgrid/config/config.yaml]: sneaked the fix in https://github.com/kubernetes/test-infra/pull/4689

**Pull request merged**